### PR TITLE
Remove several unused JS dependencies

### PR DESCRIPTION
**Pull request by hotzenklotz**
While debugging the library version conflicts for PR #7505 I ran a check for unused libs with `depcheck`. While it flagged many false positive libs I think it correctly identified these 5 libs as unused:

#### *[Name]*: *[PR where the lib was introduced]*
- `ndarray-convolve`: [Quick Select Tool PR](https://github.com/scalableminds/webknossos/pull/6542)
- `react-sortable-tree-theme-file-explore`:    [Dataset Folder PR](https://github.com/scalableminds/webknossos/pull/6591)
- `strip-ansi`: [Virtual VX Logs](https://github.com/scalableminds/webknossos/pull/6579)
- `@use-it/interval`: [Recent Datasets conversion on Dashboard PR](https://github.com/scalableminds/webknossos/pull/5597)
- `cross-fetch`: [PR to Fix airbrake-js](https://github.com/scalableminds/webknossos/pull/3859)

Hopefully this decreases our bundle size a little bit and helps to avoid version conflicts in the future.

Command to detect unused libs.
```
npx depcheck . --specials=eslint,webpack,prettier,typescript,less,esbuild
```

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Delete `node_modules` and `yarn install`
- I did tests to verify that:
  - That throwing an Error will trigger a Airbrake notifications
  - That quick select works in AI and non-AI mode
  - That VX logs can still be viewed
  - That skeletons & segment tree components looked "normal"

### Issues:
- None

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -166,7 +165,6 @@
     "classnames": "^2.2.5",
     "color-hash": "^2.0.1",
     "comlink": "^4.3.0",
-    "cross-fetch": "^3.1.5",
```
Comment (hotzenklotz) [inline package.json:169]: This lib was introduced to added a polyfill for `airbrake-js`. We have switched to `@airbrake/browser` in the meantime, which seems to work without polyfills.

**Comment by hotzenklotz** [comment]
@philippotto @daniel-wer I already tried to verify as much as possible that we really do not use these libs anywhere in our code or tooling. Yet, I didn't remember why we added some of this in the first-place or why it has become unused. Maybe some of the lib names mean more to you and you can verify that this is safe to remove. Thanks.

File: package.json [hunk 2]
```diff
@@ -154,7 +154,6 @@
     "@types/pixelmatch": "^5.2.4",
     "@types/pngjs": "^6.0.1",
     "@types/three": "^0.142.0",
-    "@use-it/interval": "^1.0.0",
```
Comment (hotzenklotz) [inline package.json:157]: We have implemented this ourselves in `libs/react_helpers`

**Comment by philippotto** [comment]
I didn't test this PR (your performed testing looks very good :+1:), but your explanations sound very reasonable and I agree that these packages seem to be unused.

> Hopefully this decreases our bundle size a little bit [...]

I would have hoped that webpack wouldn't include unused bits, but removing unused packages is still a good idea, obviously.

**Comment by hotzenklotz** [comment]
@daniel-wer Do these libs mean anything to you? Do you see any red flags?

**Comment by daniel-wer** [comment]
> @daniel-wer Do these libs mean anything to you? Do you see any red flags?

I don't see any red flags. Thanks for the detailed description and researching where these libs were introduced!

**Review by philippotto — APPROVED** [review]
cool, then, let's do it :tada:

**Pull request merged**